curriculum/js1/6.js

```diff
@@ -5,8 +5,21 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, sum = 0, orig = a) => {
+  n = a - 1;
+  if (n === 2) {
```

Comment:
I'm not sure what you are trying to do here. Could you explain or find a way to do this a simpler way?